send2cgeo.user.js

```diff
@@ -5,19 +5,7 @@
 // @author         c:geo team and contributors
 // @grant          none
 // @require        http://code.jquery.com/jquery-3.4.1.min.js
-// @include        https://www.geocaching.com/play/search*
-// @include        https://www.geocaching.com/play/search/*
-// @include        http://www.geocaching.com/seek/cache_details*
-// @include        https://www.geocaching.com/seek/cache_details*
-// @include        https://www.geocaching.com/map/*
-// @include        https://www.geocaching.com/play/map*
-// @include        https://www.geocaching.com/play/map/*
-// @include        http://www.geocaching.com/geocache/*
-// @include        https://www.geocaching.com/geocache/*
-// @include        http://www.geocaching.com/my/recentlyviewedcaches*
-// @include        https://www.geocaching.com/my/recentlyviewedcaches*
-// @include        http://www.geocaching.com/seek/nearest*
-// @include        https://www.geocaching.com/seek/nearest*
+// @include        http*://www.geocaching.com/*
```

Comment:
please add a line simular to gclh:
~~~
// @exclude          /^https?://www\.geocaching\.com/(login|jobs|careers|brandedpromotions|promotions|blog|help|seek/sendtogps|profile/profilecontent)/
~~~
(this line is from gclh, please check it before integrating)